cluster-api-1.9.yaml

```diff
@@ -0,0 +1,143 @@
+        - cluster-api-kubeadm-bootstrap-controller=${{package.full-version}}
+    pipeline:
+      - uses: go/build
+        with:
+          output: kubeadm-bootstrap-controller
+          packages: ./bootstrap/kubeadm
+          ldflags: |
+            -X sigs.k8s.io/cluster-api/version.gitVersion=${{package.version}} \
+            -X sigs.k8s.io/cluster-api/version.gitCommit=${GIT_COMMIT} \
+            -X sigs.k8s.io/cluster-api/version.gitTreeState=$(shell git diff --quiet >/dev/null 2>&1 || echo "dirty" || echo "clean") \
+            -X sigs.k8s.io/cluster-api/version.buildDate=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+
+  - name: ${{package.name}}-kubeadm-controlplane-controller
+    description: cluster-api core controller
+    dependencies:
+      provides:
+        - cluster-api-kubeadm-controlplane-controller=${{package.full-version}}
+    pipeline:
+      - uses: go/build
+        with:
+          output: kubeadm-controlplane-controller
+          packages: ./controlplane/kubeadm
+          ldflags: |
+            -X sigs.k8s.io/cluster-api/version.gitVersion=${{package.version}} \
+            -X sigs.k8s.io/cluster-api/version.gitCommit=${GIT_COMMIT} \
+            -X sigs.k8s.io/cluster-api/version.gitTreeState=$(shell git diff --quiet >/dev/null 2>&1 || echo "dirty" || echo "clean") \
+            -X sigs.k8s.io/cluster-api/version.buildDate=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+
+  - name: ${{package.name}}-clusterctl
+    description: cluster-api command line tool
+    dependencies:
+      provides:
+        - clusterctl=${{package.full-version}}
+    pipeline:
+      - uses: go/build
+        with:
+          output: clusterctl
+          packages: ./cmd/clusterctl
+          ldflags: |
+            -X sigs.k8s.io/cluster-api/version.gitVersion=${{package.version}} \
+            -X sigs.k8s.io/cluster-api/version.gitCommit=${GIT_COMMIT} \
+            -X sigs.k8s.io/cluster-api/version.gitTreeState=$(shell git diff --quiet >/dev/null 2>&1 || echo "dirty" || echo "clean") \
+            -X sigs.k8s.io/cluster-api/version.buildDate=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+    test:
+      pipeline:
+        - runs: |
+            clusterctl version | grep -q ${{package.version}}
+            clusterctl --help | grep -q "Get started with Cluster API using clusterctl"
+            clusterctl config repositories | grep helm
+
+test:
+  environment:
+    contents:
+      packages:
+        - ${{package.name}}-controller
+        - ${{package.name}}-kubeadm-bootstrap-controller
+        - ${{package.name}}-kubeadm-controlplane-controller
+        - ${{package.name}}-clusterctl
+        - curl
+  pipeline:
+    - uses: git-checkout
+      with:
+        expected-commit: 813e90d8daef60b715764ce4729b40d82f811cc1
+        repository: https://github.com/kubernetes-sigs/cluster-api
+        tag: v${{package.version}}
+    - uses: test/kwok/cluster
+    - runs: |
+        kubectl apply -f config/crd/bases/
+    - uses: test/daemon-check-output
+      with:
```

Comment:
```suggestion
    - uses: test/daemon-check-output
      with:
        setup: kubectl apply -f config/crd/bases/
```

Reply:
As the crd apply step is needed by the other daemon check output steps, we agreed to keep it separate